[Bug] vscode.npm拡張機能のアクティベーション失敗 - find-yarn-workspace-rootモジュール欠落

## 概要

`npm run tauri:dev` でのローカル起動時に、ビルトイン拡張機能 `vscode.npm` のアクティベーションが `find-yarn-workspace-root` モジュールが見つからないために失敗する。

## 再現手順

1. `npm run tauri:dev` でアプリケーションをローカル起動
2. npm拡張機能のアクティベーションが試行される
3. モジュール解決エラーで失敗

## エラーログ

```
[Error] Activating extension 'vscode.npm' failed: Cannot find module '../node_modules/find-yarn-workspace-root'
Require stack:
- /Users/maedatakurou/work/dev/vscodeee/.build/extensions/npm/out/preferred-pm.js
- /Users/maedatakurou/work/dev/vscodeee/.build/extensions/npm/out/tasks.js
- /Users/maedatakurou/work/dev/vscodeee/.build/extensions/npm/out/commands.js
- /Users/maedatakurou/work/dev/vscodeee/.build/extensions/npm/out/npmMain.js
- /Users/maedatakurou/work/dev/vscodeee/out/vs/workbench/api/node/extHostExtensionService.js
```

## 根本原因の推定

`.build/extensions/npm/` 配下のビルド成果物に `find-yarn-workspace-root` の依存パッケージが含まれていない。ビルトイン拡張機能のビルドプロセスで `node_modules` の依存解決が不完全な可能性がある。

## 影響範囲

- npm scripts の検出・実行機能が動作しない
- `package.json` のタスク検出が無効
- npm関連のコマンドパレットアクションが利用不可

## 関連ファイル

- `extensions/npm/package.json` - 依存パッケージ定義
- `.build/extensions/npm/` - ビルド成果物
- `extensions/npm/src/preferred-pm.ts` - エラー発生元のソース

## 深刻度: High

npm/yarn/pnpmプロジェクトのタスク実行・管理機能が使えなくなる。

--- a/src-tauri/src/commands/extensions.rs
+++ b/src-tauri/src/commands/extensions.rs
@@ -57,20 +57,25 @@ fn resolve_extensions_dir(app_handle: &tauri::AppHandle) -> Option<PathBuf> {
     use tauri::Manager;
 
     // Try CWD-based resolution first (works during `cargo tauri dev`).
-    // During development, the packaged extensions live in .build/extensions/.
-    // Fall back to the raw extensions/ source directory if .build/ doesn't exist
-    // (e.g., before `package-extensions` has been run).
+    // During development, prefer the source `extensions/` directory over
+    // `.build/extensions/` because the source directory contains `node_modules/`
+    // required by Language Server extensions (markdown, css, etc.) to fork
+    // child processes. `.build/extensions/` only has compiled JS but lacks
+    // `node_modules/`, causing Language Server processes to crash immediately.
+    // See: https://github.com/j4rviscmd/vscodeee/issues/283
     if let Some(dir) = std::env::current_dir()
         .ok()
-        .map(|cwd| cwd.join("../.build/extensions"))
+        .map(|cwd| cwd.join("../extensions"))
         .and_then(|p| p.canonicalize().ok())
         .filter(|p| p.is_dir())
     {
         return Some(dir);
     }
+    // Fall back to .build/extensions/ if source directory is not available
+    // (e.g., in CI or stripped source environments).
     if let Some(dir) = std::env::current_dir()
         .ok()
-        .map(|cwd| cwd.join("../extensions"))
+        .map(|cwd| cwd.join("../.build/extensions"))
         .and_then(|p| p.canonicalize().ok())
         .filter(|p| p.is_dir())
     {

